BREAKING.md

```diff
@@ -20,8 +20,13 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 ## 2.0.0-internal.3.0.0 Upcoming changes
 
 ## 2.0.0-internal.3.0.0 Breaking changes
+- [Make existing flag required](#make-existing-flag-required)
 - [Remove iframe-driver](#remove-iframe-driver)
 
+### Make existing flag required
+The `existing` flag has been updated to be expected and required in the `instantiateRuntime` method. The vast
+majority of clients can rely on the `existing` flag being there as it was introduced in client version 0.44. However, it's
+best practice to asser that 'existing' is not undefined and to use the value passed in.
```

Comment:
This description is very vague and does not tell anything interesting. It should either say that this flag was added as optional in version x and is now made required. Or, it should explain what existing flag is and how it's supposed to be used.